README.md

```diff
@@ -42,14 +42,16 @@ If these instructions don't work for you or are incomplete, please file an issue
 
 The build uses [Stack](http://docs.haskellstack.org/). If you don't already have it installed, [follow the install instructions](http://docs.haskellstack.org/en/stable/README.html#how-to-install) for your platform.  (Hint: `brew update && brew install stack`)
 
+If you have not set up the Haskell toolchain before and are trying to contribute to Unison on an M1 Mac, we have [some tips specifically for you](docs/m1-mac-setup-tips.markdown/new).
```

Comment:
@rlmark  I'm not sure what the `/new` at the end is meant for, but the link doesn't seem to work in the current README on trunk 😞